common/components/Footer/index.tsx

```diff
@@ -219,6 +220,7 @@ export default class Footer extends React.PureComponent<Props, State> {
           </div>
         </footer>
 
+        <QrSignerModal />
```

Comment:
Would prefer this component only be rendered where it contextually makes sense, somewhere in the send form.